src/torchmetrics/detection/mean_ap.py

```diff
@@ -186,7 +186,7 @@ class MeanAveragePrecision(Metric):
 
         iou_type:
             Type of input (either masks or bounding-boxes) used for computing IOU. Supported IOU types are
-            ``"bbox"`` or ``"segm"`` or both as a tuple.
+            ``"bbox"``, ``"segm"``  or both as a tuple or ``"keypoints"``.
```

Comment:
just to be sure, it should not be possible to set `iou_type=("bbox", "keypoints")`?